public/contextMenu.js

```diff
@@ -0,0 +1,409 @@
+/*
```

Comment:
Did you make any changes from the original?

Reply:
Yes, I removed the features that requires new dependencies to be installed.
Basically commented lines 21 to 23 and then commented every code instance that used them.

Reply:
A new cleanup was made, removing many features we didn't need.
Our version is now significantly different from the original.